apis/shared-dimensions/lib/resource.ts

```diff
@@ -94,7 +96,10 @@ export async function resourceQuery(id: NamedNode, graph: NamedNode, client: Par
 }
 
 export function deleteShapesQuery(id: NamedNode, graph: NamedNode) {
-  return WITH(graph, DELETE`?s ?p ?o`.WHERE`${resourceShapePatterns(id, true)}`)
+  return WITH(graph, DELETE`?shape ?p ?o .`.WHERE`
+    ${resourceShapePatterns(id, true)}
+    ?shape ?p ?o .
```

Comment:
Because the `resourceShapePatterns` function now does not include the `?s ?p ?o` pattern, that has to be added where necessary

Reply:
Sounds good, and is clear as the VAR Name is called ?shape. And it is restricted on the specific graph.

(I feel at some point we also need SHACL shapes for delete operations, this here seems a bit dangerous a times.)

Reply:
Not sure if I understood the last remark. Could you elaborate?